tanner/emulators/php_object_injection.py

```diff
@@ -12,6 +12,11 @@ def __init__(self, loop=None):
         self.helper = PHPSandboxHelper(self._loop)
 
     async def get_injection_result(self, code):
+        """
+        Injects the code from attacker to vulnerable code and get emulation results from php sandbox.
+        :param code (str): Input payload from attacker
+        :return: object_injection_result (dict): Keys - file_md5 (md5 hash), stdout (injection result)
```

Comment:
no need to write the name of the returned variable, the type and key description are enough